types/artillery/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false
```

Comment:
Don't see where you're getting this error?

Reply:
yep sorry good catch. I had interfaces and switched them to types but forgot to remove this rule.